### DISCUSSION TITLE
repeat with overlap

### DISCUSSION THREAD

**Discussion by Cdriko** [shader-park/shader-park-core#84] [General]
Hello I try to repeat assemblies with a overlaping space, using the repeat function
[in this example](https://shaderpark.com/sculpture/-N5Ea0Pyb_s5SauCQRk2).
But the ovelapping parts of space is not rendered. Any idea to solve this problem ?
Maybe using another function?
Thanks

**Comment by PWhiddy** [comment]
reposting from issue #83 

You’re correct for noticing the cells when using repeat can’t overlap. What I recommend is doubling the spacing between the repeated assemblies, and then rendering multiple repeated instances offset to fill the space. For things which are repeated on all three axes would require quite a few extra draws though. That’s the best solution i think of at this moment!

Trying to think if there’s an easier or more efficient way. Really if the cells are overlapping, you want to be taking the union between “separate” sdfs which each need to be evaluated. Maybe there’s a way to create a helper function for this..
